File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.js
function solution(arr1, arr2) {
    if (arr1.length > arr2.length) return 1;
    else if (arr1.length < arr2.length) return -1;
    
    let a = 0;
    let b = 0;
    for (let i = 0; i < arr1.length; i++) {
        a += arr1[i];
        b += arr2[i];
    }
    
    if (a > b) return 1;
    else if (a < b) return -1;
    else return 0;
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    if len(arr1) > len(arr2):
        return 1
    elif len(arr2) > len(arr1):
        return -1
    result1, result2 = 0, 0
    for i in range(0,len(arr1)):
        result1 += arr1[i]
        result2 += arr2[i]
    return 1 if result1 > result2 else -1 if result1 < result2 else 0